kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -83,13 +82,9 @@ abstract class KyuubiOperation(session: Session) extends AbstractOperation(sessi
             MetricRegistry.name(OPERATION_FAIL, opType, errorType)))
           val ke = e match {
             case kse: KyuubiSQLException => kse
-            case te: TTransportException
-                if te.getType == TTransportException.END_OF_FILE &&
-                  StringUtils.isEmpty(te.getMessage) =>
-              // https://issues.apache.org/jira/browse/THRIFT-4858
-              KyuubiSQLException(
-                s"Error $action $opType: Socket for ${session.handle} is closed",
-                e)
+            case te: TTransportException =>
+              warn(s"Error $action $opType: Socket for ${session.handle} is closed", e)
+              KyuubiSQLException.connectionClosed(session.handle, e)
```

Comment:
`socket closed` may be more appropriate. and cc @yaooqinn, it was introduced in #375.

Reply:
I don't have a strong option here, both seem fine to me